3p/beop.js

```diff
@@ -101,24 +104,13 @@ function getBeOpinionAsyncInit(global, accountId) {
         }
       },
       onHeightChange: function(newHeight) {
-        const c = global.document.getElementById('c');
-        const boundingClientRect = c./*REVIEW*/ getBoundingClientRect();
-        context.requestResize(boundingClientRect.width, newHeight);
+        if (resizeRequestTimeout != null) {
+          clearTimeout(resizeRequestTimeout);
+        }
+        context.requestResize(undefined, newHeight);
```

Comment:
Is the idea here that we only resize vertically? Did you test to make sure this works? Might need to pass the existing width, but not sure.